internal/provider/resource_k8s.go

```diff
@@ -837,14 +837,14 @@ func (k *K8sDeployer) removeDeletedContracts(ctx context.Context, sub *substrate
 	return nil
 }
 
-func getNodeIdByCapacityId(sub *substrate.Substrate, capacityId uint64) (uint32, error) {
-	contract, err := sub.GetContract(capacityId)
-	if err != nil {
-		return 0, errors.Wrapf(err, "couldn't get capacity reservation contract with Id: (%d)", capacityId)
-	}
-	nodeId := uint32(contract.ContractType.CapacityReservationContract.NodeID)
-	return nodeId, nil
-}
+// func getNodeIdByCapacityId(sub *substrate.Substrate, capacityId uint64) (uint32, error) {
+// 	contract, err := sub.GetContract(capacityId)
+// 	if err != nil {
+// 		return 0, errors.Wrapf(err, "couldn't get capacity reservation contract with Id: (%d)", capacityId)
+// 	}
+// 	nodeId := uint32(contract.ContractType.CapacityReservationContract.NodeID)
+// 	return nodeId, nil
+// }
 
 func (k *K8sDeployer) updateFromRemote(ctx context.Context, sub *substrate.Substrate) error {
```

Comment:
this function while returning error interface, it swallows the errors with prints